packages/common/src/lib/core/services/test.service.ts

```diff
@@ -11,12 +11,14 @@ import { TestMapper } from './mappers/test.mapper';
 export class TestService {
   constructor(private readonly testMapper: TestMapper) {}
 
-  public getList(): Observable<Test[]> {
+  getList(): Observable<Test[]> {
```

Comment:
I know you want to try new things, but methods without any access modifier are confusing for me. Anyways, just my opinion.